app/controllers/review_controller.rb

```diff
@@ -14,9 +30,9 @@ def approve_2i_review
       reviewer_id: nil,
       status: status,
     )
-      generate_change_note("2i approved")
+      generate_change_note("2i approved", params[:additional_comment])
 
-      redirect_to step_by_step_page_path(@step_by_step_page.id), notice: "Step by step page was successfully approved_2i."
+      redirect_to step_by_step_page_path(@step_by_step_page.id), notice: "Step by step page was successfully 2i approved. Please let the author know."
     end
```

Comment:
Did you want this to show the submit_2i_verdict view if the update fails?

Reply:
Good spot. I need an `else` in there.